src/views/stories/index.tsx

```diff
@@ -15,28 +18,44 @@ import StoriesCard from 'views/communities/StoriesCard';
 function StoriesScreen() {
     const navigation = useNavigation();
     const dispatch = useDispatch();
+    // const [reachedEndList, setReachedEndList] = useState(false);
+    // const [storiesOffset, setStoriesOffset] = useState(0);
+
+    const stories = useSelector(
+        (state: IRootState) => state.stories.stories.data
+    );
+    const storiesCount = useSelector(
+        (state: IRootState) => state.stories.stories.count
+    );
 
-    const stories = useSelector((state: IRootState) => state.stories.stories);
     const refreshing = useSelector(
         (state: IRootState) => state.stories.refreshing
     );
 
     useEffect(() => {
-        dispatch(addStoriesToStateRequest(0, 5));
+        //TODO: improve pagingEnabled list
```

Comment:
yes, let's do it now and not use `stories.count` here.
ideally, we could load in batches of ~21. Each row has 3 columns and the biggest screens may have space only for 5 rows. Loading in batches of 21 until the data.length is smaller than 21 (which means, they have reached an end) seems to be a good solution to me. But feel free to pick another approach.
Request again review once changed.